models/data_warehouse.model.lkml

```diff
@@ -2085,12 +2085,6 @@ explore: server_fact {
     relationship: one_to_many
   }
 
-  join: account {
```

Comment:
@sri-byte - is this the only place we use account? Are there other places to remove it from?

Reply:
There are other places where account is used, but I've never had any requests to change/update those explores.